.travis.yml

```diff
@@ -1,11 +1,15 @@
 sudo: false
 language: node_js
 node_js:
-  - '0.12'
-  - '0.10'
-  - iojs
+  - 4
+  - "0.10"
+  - "stable"
+
+before_install:
+  - "npm config set progress=false"
```

Comment:
I don't think there are any benefits for Travis (and I doubt whether there's significant ones locally). Best not to have invocations like this in the code, it'll only lead to more questions in the future.